site/_includes/partials/footer.njk

```diff
@@ -90,7 +90,7 @@
.773.58 1.667.58 2.68 0 .12-.006.24-.02.36-.013.12-.02.187-.02.2h-7.64c.067.907.394 1.607.98 2.1.587.493 1.254.74 2 .74 1.147 0 2.027-.533 2.64-1.6l1.88.9a5.192 5.192 0 01-1.84 1.96c-.773.48-1.686.72-2.74.72zm2.52-6.5a2.402 2.402 0 00-.32-.98c-.186-.333-.48-.613-.88-.84-.386-.24-.873-.36-1.46-.36-.653 0-1.22.2-1.7.6-.466.4-.786.927-.96 1.58h5.32zm4.451-8.14h2.2V15h-2.2V.68zm9.509 14.64c-.986 0-1.873-.227-2.66-.68a5.098 5.098 0 01-1.86-1.9c-.44-.8-.66-1.7-.66-2.7 0-1 .22-1.9.66-2.7a4.949 4.949 0 011.86-1.88c.787-.467 1.674-.7 2.66-.7 1 0 1.894.233 2.68.7.787.453 1.4 1.08 1.84 1.88.44.8.66 1.7.66 2.7 0 1-.22 1.9-.66 2.7a4.937 4.937 0 01-1.84 1.9c-.786.453-1.68.68-2.68.68zm0-2.02c.534 0 1.027-.133 1.48-.4.454-.267.82-.647 1.1-1.14.28-.493.42-1.067.42-1.72 0-.667-.14-1.247-.42-1.74a2.864 2.864 0 00-1.1-1.12 2.865 2.865 0 00-1.48-.4c-.52 0-1.013.133-1.48.4a2.87 2.87 0 00-1.1 1.12c-.266.493-.4 1.073-.4 1.74 0 .653.14 1.227.42 1.72.28.493.647.873 1.1 1.14.454.267.94.4 1.46.4zm7.222-8.22h2.08v1.28h.12c.28-.453.7-.833 1.26-1.14.573-.307 1.22-.46 1.94-.46.853 0 1.64.227 2.36.68.733.453 1.313 1.087 1.74 1.9.44.8.66 1.7.66 2.7 0 1-.22 1.9-.66 2.7a4.992 4.992 0 01-1.74 1.9c-.72.453-1.507.68-2.36.68-.72 0-1.367-.153-1.94-.46-.56-.307-.98-.693-1.26-1.16h-.12l.12 1.4v4.22h-2.2V5.08zm5.02 8.22c.52 0 1-.133 1.44-.4a2.88 2.88 0 001.08-1.14c.28-.507.42-1.08.42-1.72 0-.64-.14-1.207-.42-1.7a2.849 2.849 0 00-1.08-1.16c-.44-.267-.92-.4-1.44-.4a2.83 2.83 0 00-1.46.4c-.44.267-.8.647-1.08 1.14-.267.493-.4 1.067-.4 1.72s.133 1.227.4 1.72c.28.493.64.873 1.08 1.14.453.267.94.4 1.46.4zm11.912 2.02c-.974 0-1.854-.227-2.64-.68a4.834 4.834 0 01-1.82-1.88c-.44-.813-.66-1.72-.66-2.72 0-.96.206-1.84.62-2.64a4.844 4.844 0 011.78-1.92c.76-.48 1.626-.72 2.6-.72 1 0 1.866.22 2.6.66a4.286 4.286 0 011.7 1.84c.386.773.58 1.667.58 2.68 0 .12-.007.24-.02.36a3.41 3.41 0 00-.02.2h-7.64c.066.907.393 1.607.98 2.1a3.02 3.02 0 002 .74c1.146 0 2.026-.533 2.64-1.6l1.88.9a5.2 5.2 0 01-1.84 1.96c-.774.48-1.687.72-2.74.72zm2.52-6.5a2.433 2.433 0 00-.32-.98c-.187-.333-.48-.613-.88-.84-.387-.24-.874-.36-1.46-.36-.654 0-1.22.2-1.7.6-.467.4-.787.927-.96 1.58h5.32zm4.45-3.74h2.08v1.48h.12c.214-.507.6-.927 1.16-1.26.56-.347 1.134-.52 1.72-.52.48 0 .874.067 1.18.2v2.3a3.35 3.35 0 00-1.52-.34c-.72 0-1.326.28-1.82.84-.48.547-.72 1.22-.72 2.02V15h-2.2V5.08zm11.441 10.24c-1.134 0-2.074-.26-2.82-.78-.734-.533-1.254-1.2-1.56-2l1.96-.84c.226.547.553.96.98 1.24.44.28.92.42 1.44.42.546 0 .993-.1 1.34-.3.346-.213.52-.487.52-.82 0-.32-.14-.58-.42-.78-.28-.213-.74-.4-1.38-.56l-1.36-.32a4.5 4.5 0 01-1.94-.96c-.534-.48-.8-1.1-.8-1.86 0-.92.38-1.647 1.14-2.18.76-.547 1.693-.82 2.8-.82.92 0 1.733.193 2.44.58.72.387 1.24.947 1.56 1.68l-1.94.82a1.87 1.87 0 00-.86-.92c-.387-.2-.807-.3-1.26-.3-.44 0-.827.1-1.16.3-.334.187-.5.44-.5.76 0 .28.113.507.34.68.24.173.613.32 1.12.44l1.48.36c1 .253 1.746.64 2.24 1.16.493.507.74 1.12.74 1.84 0 .587-.18 1.12-.54 1.6-.347.48-.834.86-1.46 1.14-.614.28-1.314.42-2.1.42z" fill="#5F6368"/></svg>
```

Comment:
Should this aria-label be localized?

Reply:
No, probably not since it's a brand and the SVG image literally has the words in it.